syntax = "proto2";
package maps.analyzer.common.tests.proto;

message MockInnerData
{
    enum Id { ID = 1; }
    optional int32 field_int = 1;
    optional string field_str = 2;
}

message MockData
{
    enum Id { ID = 2; }
    optional MockInnerData inner_data = 1;
}
